packages/components/organisms/f-checkout/src/services/orderPlacementServicesApi.js

```diff
@@ -0,0 +1,20 @@
+import axios from 'axios';
+import { version as applicationVersion } from '../../package.json';
+
+export default {
+    async placeOrder (url, data, timeout, state) {
+        const authHeader = state.authToken && `Bearer ${state.authToken}`;
+
+        const config = {
+            headers: {
+                'Content-Type': 'application/json;v=2',
+                'x-je-application-id': 7, // Responsive Web
+                'x-je-application-version': applicationVersion,
+                Authorization: authHeader
```

Comment:
Do you need to conditionally add this like you have done in `CheckoutServiceApi.js`?

Reply:
same as above :)